dist/eModal.js

```diff
@@ -452,12 +452,22 @@
             }
 
             var html = ('<div class=modal-body style="position: absolute;width: 100%;background-color: rgba(255,255,255,0.8);height: 100%;">%1%</div>' +
-                '<iframe class="embed-responsive-item" frameborder=0 src="%0%" style="width:100%;height:75vh;display:block;"/>')
+                    '<iframe class="embed-responsive-item" frameborder=0 src="%0%" style="width:100%;height:75vh;display:block;"/>')
                 .replace('%0%', params.message || params.url || params)
                 .replace('%1%', defaultSettings.loadingHtml);
 
-            var message = $(html)
-                .load(iframeReady);
+            var message;
```

Comment:
Very hard to read condition, it requires context about we are doing here!
Can we create a less expensive function and use it.
Eg:

```
if (minVersion('3.0.0')) {...} else {...}

function minVersion(version) {
    const [$major, $minor, $patch] = $.fn.jquery.split('.');
    const [major, minor, patch] = version.split('.');

    return !(
        (major > $major) ||
        (major === $major && minor > $minor) ||
        (major === $major && minor === $minor && patch > $patch)
    );
}

```